### PR TITLE
Make RC protocol being decoded part of each opened log

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

**Comment by peterbarker** [comment]
Tested in SITL and on my Bixler.

**Comment by khancyr** [comment]
if it survive the Bixler, it should be fine !

File: libraries/AP_HAL_ChibiOS/RCInput.h [hunk 1]
```diff
@@ -49,7 +49,9 @@ class ChibiOS::RCInput : public AP_HAL::RCInput {
     int16_t get_rssi(void) override {
         return _rssi;
     }
-        
+
+    const char *protocol() const { return last_protocol; }
```
Comment (OXINARF) [inline libraries/AP_HAL_ChibiOS/RCInput.h]: Missing override. Applies to all HALs.

**Comment by WickedShell** [comment]
> if it survive the Bixler, it should be fine !

I'm not sure that's the conclusion I'd draw :)

**Comment by peterbarker** [comment]
Merged after discussions.

**Pull request merged**